lib/dynamo-backup.js

```diff
@@ -158,9 +158,15 @@ DynamoBackup.prototype._copyTable = function (tableName, itemsReceived, callback
         if (err) {
             return callback(err);
         }
+        var limit = 1;
 
-        var readPercentage = self.readPercentage;
-        var limit = Math.max((data.Table.ProvisionedThroughput.ReadCapacityUnits * readPercentage) | 0, 1);
+        if (data.Table.BillingModeSummary.BillingMode == "PAY_PER_REQUEST") {
```

Comment:
this is the main bit we care about

Reply:
Do we want PAY_PER_REQUEST in a constant?